pop-api/src/v0/fungibles/traits.rs

```diff
@@ -101,10 +106,13 @@ pub trait Psp22Metadata {
 pub trait Psp22Mintable {
 	/// Creates `value` amount of tokens and assigns them to `account`, increasing the total supply.
 	///
+	/// The selector for this message is `0x7a9da510` (first 4 bytes of
+	/// `blake2b_256("PSP22Burnable::burn")`).
+	///
 	/// # Parameters
 	/// - `account` - The account to be credited with the created tokens.
 	/// - `value` - The number of tokens to mint.
-	#[ink(message)]
+	#[ink(message, selector = 0xfc3c75d4)]
```

Comment:
This doesnt match the comment, think you have them switched with burn.

Reply:
The comment was wrong. Resolved